viewer/packages/camera-manager/src/StationaryCameraManager.ts

```diff
@@ -27,6 +27,7 @@ export class StationaryCameraManager implements CameraManager {
   private readonly _minFOV: number;
   private readonly _stopEventTrigger: DebouncedCameraStopEventTrigger;
   private _isDragging = false;
+  private _downEventCache: Array<PointerEvent> = [];
```

Comment:
Concider using a pointerId -> Vector2 (clientX, clientY). Would simplify the usage a little since you would not have to scan the list all the time. Also you could use built in functionality for length between two points.

Reply:
No. Well, I did try with the Map and although some things become simpler I struggle to find the direction of the zoom as I was depending on the order of the down events. Also, we now technically support zooming and drag at the same time, which I was not able to do. So I kinda like my current solution better.